packages/manager/apps/telecom-task/cypress/support/po/base.js

```diff
@@ -0,0 +1,32 @@
+export default class BasePage {
+  constructor() {
+    this.mainSection = 'body';
+    this.hashUrl = '#!/';
```

Comment:
Does the prefix shouldn't be removed instead? (https://docs.angularjs.org/api/ng/provider/$locationProvider#hashPrefix)

Reply:
Relative to `manager-layout-ovh` which is not used in `telecom-task`.

Reply:
Otherwise does the `@ovh-ux/manager-core` should be responsible of that configuration?

Reply:
IMO, this is the responsibility of the main app, not of one module loaded in an app.